pkg/quota/api/types.go

```diff
@@ -54,6 +54,30 @@ type ClusterResourceQuotaList struct {
 	Items []ClusterResourceQuota
 }
 
+// AppliedClusterResourceQuota mirrors ClusterResourceQuota at a namespace scope, for projection
+// into a namespace
+type AppliedClusterResourceQuota struct {
+	unversioned.TypeMeta
+	// Standard object's metadata.
+	kapi.ObjectMeta
+
+	// Spec defines the desired quota
+	Spec ClusterResourceQuotaSpec
```

Comment:
definitely want the spec as well... the UI will use the spec to show things like "cluster quota limit of 1000 pods, this project is using 5, other projects are using 900"